app/Http/Controllers/PostsController.php

```diff
@@ -144,8 +144,8 @@ public function update(Request $request, Post $post)
             'quantity' => 'nullable|integer',
         ]);
 
-        // Only allow an admin or the user who owns the post to update.
-        if (token()->role() === 'admin' || auth()->id() === $post->northstar_id) {
+        // Only allow an admin/staff or the user who owns the post to update.
+        if (in_array(token()->role, ['admin', 'staff']) || auth()->id() === $post->northstar_id) {
```

Comment:
Maybe out of the scope of this...but I wonder if there should be a little like `isAllowed()` helper. Cause I see us doing these checks a lot of places. And that helper would also check client scopes.

Reply:
Or if something else make more sense, then we could do that...just trying to centralize this logic a little more.